tsconfig.json

```diff
@@ -31,6 +31,7 @@
         "moduleResolution": "node",
         "resolveJsonModule": true,
         "allowSyntheticDefaultImports": true,
+        "esModuleInterop": true,
```

Comment:
Why this change?

Reply:
It's based on a recommendation from [webpack docs](https://webpack.js.org/configuration/configuration-languages/)

Reply:
I think we don't need it because this tsconfig extends `expo/tsconfig.base` and `esModuleInterop` is already `true` there.

Reply:
Removed!